qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -38,10 +38,15 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	InvalidVirtualMethod(
 			"The virtual method `{0}` in the type `{1}` is not applicable for the base object `{2}` type."), //
 	InfixNotationParameterRequired("A parameter for the infix notation method `{0}` is required."), //
-	InvalidMethodInfixNotation("The method `{0}` cannot be used with infix notation, because it has not `1` parameter."), //
-	
+	InvalidMethodInfixNotation(
+			"The method `{0}` cannot be used with infix notation, because it has not `1` parameter."), //

```

Comment:
```suggestion
	InvalidMethodInfixNotation(
			"The method `{0}` cannot be used with infix notation, because it does not have exactly `1` parameter."), //

```

Reply:
fixed